From 01b1b6b68fa61771f8b1149724ccb255cb87a30c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 22 Sep 2022 17:47:44 +0300
Subject: [PATCH] [SW-75097] hl: zephyr: preboot: remove mbedtls

mbedtls module is no longer used by Preboot - remove it.

Change-Id: Ibed2c43675092429674c5a362028c453e91d4d6f
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index 94422f8..156468d 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -119,7 +119,7 @@
 ZEPHYR_BUILD_CMDS += && $(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/$(ZEPHYR_PREBOOT_DIR)
 ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $(@D)/$(ZEPHYR_PREBOOT_DIR)/zephyr/zephyr.bin $(BINARIES_DIR)/preboot.bin
 ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $(@D)/$(ZEPHYR_PREBOOT_DIR)/zephyr/zephyr.elf $(BINARIES_DIR)/zephyr_preboot.elf
-ZEPHYR_PREBOOT_MODULES = fdt boot_info mbedtls eeprom
+ZEPHYR_PREBOOT_MODULES = fdt boot_info eeprom
 $(foreach module,$(ZEPHYR_PREBOOT_MODULES),$(call DEFINE_MODULE_ENVVARS,$(module),$(ZEPHYR_PREBOOT_DIR),$(ZEPHYR_PREBOOT_NAME)))
 endif #BR2_PACKAGE_ZEPHYR_ARC_PREBOOT
 
